test/integration/basic_test.go

```diff
@@ -161,13 +173,14 @@ func FeatureContext(s *godog.Suite) {
 
 	s.Step(`Minishift (?:has|should have) state "([^"]*)"`, m.shouldHaveState)
 	s.Step(`Minishift should have a valid IP address`, m.shouldHaveAValidIPAddress)
-	s.Step(`executing "minishift ([^"]*)"`, m.executingCommand)
-	s.Step(`executing "oc ([^"]*)`, m.executingOcCommand)
+	s.Step(`execut[es|ing]+ "minishift ([^"]*)"`, m.executingCommand)
+	s.Step(`execut[es|ing]+ "oc ([^"]*)`, m.executingOcCommand)
```

Comment:
It should be present continuous tense (+ing)... and keep consistency (as AND is part of the same tense)